extensions/amp-story/1.0/amp-story-render-service.js

```diff
@@ -15,8 +15,17 @@
  */
 
 import {CommonSignals} from '../../../src/common-signals';
+import {Services} from '../../../src/services';
 import {whenUpgradedToCustomElement} from '../../../src/dom';
 
+/**
+ * Maximum milliseconds to wait for service to load before logging a warning.
+ * Needs to be shorter than the render delay timeout to account for the latency
+ * downloading and executing the amp-story js.
+ * @const
+ */
+const LOAD_TIMEOUT = 1500;
```

Comment:
Changing the timeout from 3000 to 1500 sounds a bit extreme. I understand that this need to be shorter than 3000, but can we do 2900? This is a bit hacky but that'll do for now, until we work on optimizing the FCP/LCP in Q1 2021.